### DISCUSSION TITLE
这周你做了什么？2022-05-27

### DISCUSSION THREAD

**Discussion by jiacai2050** [EmacsTalk/emacstalk.github.io#20] [Show and tell]
- 可以随便讨论本周你打算做什么与 Emacs 相关的事情，甚至可以寻求帮助或反馈。
- 请记住，什么都不做也没关系！
- Happy Hacking Emacs！

**Comment by jiacai2050** [comment]
- 这周工作琐事多，心有些累，不过高兴的是有个朋友主动报名了下一期的节目，非常难得，感觉在国内自我推荐的案例实在是太少了
  - https://github.com/EmacsTalk/joinus/pull/14

**Comment by alaskasquirrel** [comment]
https://github.com/Lisp-Life/Lisp-China 有意分享的朋友可以完善一下这个资源搜集项目。Emacs Lisp方面，我不太熟悉。
